.github/workflows/build.yml

```diff
@@ -15,24 +15,38 @@ jobs:
     strategy:
       matrix:
         include:
-          - os: '10.11-cross-${{github.run_id}}'
-          - os: '11-arm64-cross-${{github.run_id}}'
-          - os: 'ubuntu-latest'
+          - os: 10.11-cross-${{github.run_id}}
+          - os: 11-arm64-cross-${{github.run_id}}
+          - os: ubuntu-latest
             container: '{"image": "ghcr.io/homebrew/ubuntu22.04:master", "options": "--user=linuxbrew"}'
             workdir: /github/home
+          - os: ubuntu-22.04-arm
+            container: '{"image": "rubylang/ruby:3.3.7-jammy"}'
+            bootstrap: true
       fail-fast: false
     runs-on: ${{matrix.os}}
     container: ${{matrix.container && fromJSON(matrix.container) || ''}}
     defaults:
       run:
         working-directory: ${{matrix.workdir || github.workspace}}
     steps:
+      - name: Bootstrap
+        if: matrix.bootstrap
+        run: |
+          apt-get update
+          apt-get install -y --no-install-recommends curl git gcc libc-dev make gawk
+          mkdir -p /home/linuxbrew/.linuxbrew/bin
+          git clone https://github.com/Homebrew/brew /home/linuxbrew/.linuxbrew/Homebrew
```

Comment:
Can we use `Homebrew/setup-homebrew` for this?

Reply:
`Homebrew/setup-homebrew` does not support installing Homebrew yet. The arm64 images do not have Homebrew installed (because we don't support it).

Do we want to add that support yet to the action? Or wait until we properly support arm64?

Reply:
Was just curious if this would work.



> When we have a Docker image (which requires Portable Ruby first), we'll switch to use that in this CI rather than `git clone`. This bootstrap step is temporary for that reason.

Agreed this seems better 👍🏻